eng/Versions.props

```diff
@@ -248,6 +248,9 @@
     <IdentityServer4EntityFrameworkStoragePackageVersion>3.0.0</IdentityServer4EntityFrameworkStoragePackageVersion>
     <MessagePackPackageVersion>2.1.90</MessagePackPackageVersion>
     <MessagePackAnalyzerPackageVersion>$(MessagePackPackageVersion)</MessagePackAnalyzerPackageVersion>
+    <MicrosoftIdentityWebPackageVersion>0.2.0-preview</MicrosoftIdentityWebPackageVersion>
```

Comment:
By next week we'll have released 0.2.1-preview which will be needed for the Blazor scenarios

Reply:
Yes, these versions will need to be manually updated. Can you add an action item to update these whenever there's a new Microsoft.Identity.Web release?

Reply:
This will be updated when blazor templates are merged.